app/models/payment.rb

```diff
@@ -0,0 +1,11 @@
+class Payment < ApplicationRecord
+  belongs_to :order
+
+  enum payment_method: { credit_card: 0, mobile_banking: 1, bank_transfer: 2, cash_on_delivery: 3 }
+  enum payment_status: { pending: 0, completed: 1, failed: 2 }
```

Comment:
Also, freeze the methods and statuses and then use them

Reply:
Done bhai.